php-frontend/src/main/java/org/sonar/plugins/php/api/visitors/PHPCheck.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010 SonarSource and Akram Ben Aissi
+ * sonarqube@googlegroups.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonar.plugins.php.api.visitors;
+
+import com.google.common.annotations.Beta;
+import org.sonar.squidbridge.api.CodeVisitor;
+
+/**
+ * Marker interface for all JavaScript checks.
+ */
+@Beta
+public interface PHPCheck extends CodeVisitor {
```

Comment:
@lindamartin looks like it's not required at all. I can't even find from where it was cope/pasted. Should be fixed